### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by Jasonlovesharon**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

File: jekyll/_posts/2020-10-30-daily-message-and-chatbot.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
+---
+title: "基于wechaty的定时消息推送（可以哄女朋友）、智能聊天和私人助理"
+author: jasonlovesharon
+email: 54027901@163.com
+tags:
+  - nodejs
+  - wechaty
+  - wechaty-puppet-padplus
+
```
Comment (wj-Mcat) [inline jekyll/_posts/2020-10-30-daily-message-and-chatbot.md]: Please remove this blank line, thanks a lot.

File: jekyll/_posts/2020-10-30-daily-message-and-chatbot.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+---
+
+
+## 前言
```
Comment (wj-Mcat) [inline jekyll/_posts/2020-10-30-daily-message-and-chatbot.md]: There should be only on blank line, please change it, thanks a lot.

File: jekyll/_posts/2020-10-30-daily-message-and-chatbot.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+  return year + '-' + month + '-' + day + '日 ' + hour + ':' + min + ' ' + str
+}
+export { getDay, formatDate }
+```
+
+![关键字触发消息](/assets/2020/daily-message-and-chatbot/keyword_push)
+![无法回答推送名片](/assets/2020/daily-message-and-chatbot/push_contact)
+![定时推送消息](/assets/2020/daily-message-and-chatbot/daily_message)
+![智能聊天](/assets/2020/daily-message-and-chatbot/talk_to_bot)
+
```
Comment (wj-Mcat) [inline jekyll/_posts/2020-10-30-daily-message-and-chatbot.md]: I think that you have forgot the suffix of image file. Is that right ?

File: jekyll/_posts/2020-10-30-daily-message-and-chatbot.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+## 后记
+
+到此已经实现了私人人助理和定时推送消息的功能（[项目地址](https://github.com/jasonlovesharon/my_bot)），由于对Typescript不熟，很多功能都是借鉴很多大神的轮子实现的，还是对Python熟悉点，下步准备用Python来实现以上功能，并进一步拓展更多有趣的功能。
```
Comment (wj-Mcat) [inline jekyll/_posts/2020-10-30-daily-message-and-chatbot.md:368]: Please add a new line at the end of the post, thanks a lot.

**Review by wj-Mcat — CHANGES_REQUESTED** [review]
There are some small changes that you should do, Please go ahead to make all things good, so that we can merge it. 🙈

File: jekyll/_developers/jasonlovesharon.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+name: jasonlovesharon
+site: https://github.com/jasonlovesharon
+avatar: /assets/developers/jasonlovesharon/avatar.png
+bio: python developer
+email: mailto:54027901@163.com
```
Comment (huan) [inline jekyll/_developers/jasonlovesharon.md]: Remove `mailto:`

Comment (Jasonlovesharon) [reply jekyll/_developers/jasonlovesharon.md]: Thanks for your check,I have fixed all ,please help review~~

File: jekyll/_posts/2020-10-30-daily-message-and-chatbot.md (hunk 3 above)
Comment (Jasonlovesharon) [reply jekyll/_posts/2020-10-30-daily-message-and-chatbot.md]: yes,you are right,I have added, thank you..

File: jekyll/_developers/jasonlovesharon.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+name: jasonlovesharon
+site: https://github.com/jasonlovesharon
+avatar: /assets/developers/jasonlovesharon/avatar.png
+bio: python developer
+email:54027901@163.com
```
Comment (wj-Mcat) [inline jekyll/_developers/jasonlovesharon.md]: A small change: there should be a blank before your email address.  Please make another commit to make it perfect.

**Review by wj-Mcat — CHANGES_REQUESTED** [review]
A small step can make it perfect.

File: jekyll/_developers/jasonlovesharon.md (hunk 6 above)
Comment (Jasonlovesharon) [reply jekyll/_developers/jasonlovesharon.md]: got it,thanks a lot

**Comment by Jasonlovesharon** [comment]
@huan @wj-Mcat @lijiarui ......

**Comment by huan** [comment]
It's great to know that you want to use Python Wechaty to build your next bot!

Link to Wechaty Contributors Nomination: https://github.com/wechaty/PMC/issues/16

**Review by huan — APPROVED** [review]
LGTM

**Comment by Jasonlovesharon** [comment]
> It's great to know that you want to use Python Wechaty to build your next bot!
> 
> Link to Wechaty Contributors Nomination: [wechaty/PMC#16](https://github.com/wechaty/PMC/issues/16)

yep,life is short,you need Python,lol... thanks a lot for your approval...

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for sharing this great post, the blog is online with URL <https://wechaty.js.org/2020/10/30/daily-message-and-chatbot>

> You've invited Jason to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

1. I have invited you to join the Wechaty Contributor Team on our GitHub Wechaty Organization, please accept it by visiting <https://github.com/wechaty>
1. We have a WeChat room for Wechaty Contributors only, if you'd like to join, please contact @lijiarui on WeChat.

**Pull request merged**